widgets/card_widget.php

```diff
@@ -10,35 +10,35 @@
  *
  * @since 1.0.0
  */
-class Wasyo_Image_Card extends \Elementor\Widget_Base
+class Image_Card extends \Elementor\Widget_Base
 {
 
 	/**
 	 * Get widget name.
 	 *
-	 * Retrieve wasyo card widget name.
+	 * Retrieve Image card widget name.
 	 *
 	 * @since 1.0.0
 	 * @access public
 	 * @return string Widget name.
 	 */
 	public function get_name()
 	{
-		return 'wasyo_card';
+		return 'Image_card';
 	}
 
 	/**
 	 * Get widget title.
 	 *
-	 * Retrieve Wasyo-Card widget title.
+	 * Retrieve Image-Card widget title.
 	 *
 	 * @since 1.0.0
 	 * @access public
 	 * @return string Widget title.
 	 */
 	public function get_title()
 	{
-		return esc_html__('Wasyo Card', 'elementor-Wasyo-Card-widget');
+		return esc_html__('Image Card', 'elementor-Card-widget');
```

Comment:
For your benefit I'm letting you know: Inconsistent naming! The argument `'elementor-Card-widget'` should have had no capital letters.

The bigger problem with this, from: https://codex.wordpress.org/I18n_for_WordPress_Developers#Text_Domains:

  The text domain must match the “slug” of the plugin.

Of course, you must specify the Text Domain of the plugin, which you haven't. You should remove Text Domain and not have it at all. Doing so would remove all my objections about Text Domain.